.env.example

```diff
@@ -17,6 +17,8 @@ DATABASE_USER= postgres #example:postgres
 DATABASE_PASSWORD=password
```

Comment:
Remove values to these variables.
Eg: DATABASE_PASSWORD=
      PRODUCTION_DATABASE=
      JWT_SECRET=